origin-discovery/src/apollo/db.js

```diff
@@ -27,7 +27,8 @@ function _makeListing (row) {
     price: row.data.price,
     commission: row.data.commission,
     commissionPerUnit: row.data.commissionPerUnit,
-    display: listingMetadata.getDisplay(row.id)
+    display: listingMetadata.getDisplay(row.id),
+    marketplacePublisher: row.data.ipfs.marketplacePublisher
```

Comment:
marketplacePublisher was added to the Listing model so rather than referencing the IPFS data, I would reference the model data. Change this line to:
`marketplacePublisher: row.data.marketplacePublisher`